website/blog/2025-09-29-blog-post-title.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Bringing NumPy's type-completeness score to nearly 90%
```

Comment:
"How we doubled type completeness in NumPy with a one-line fix"

Reply:
thanks for the suggestion, but this covers more than the one-line fix so I feel like it would be reductive to have that be the title